### PR TITLE
Feature/css partials

### PR DESCRIPTION
**Pull request by DougieDev**
###### What's this PR do?
- Refactor class properties in accordance with scss, disable unused scss, remove and import variables from partial file
- Create partial file for Scss variables
- Refactor class elements in index.html and page-manip.js to be more semantic and identifiable

###### Where should the reviewer start?
- css/_variables.scss
- style.scss
- index.html
- page-manipulation.js

###### How should this be manually tested?
- run npm start and ensure application is still operational

###### Any background context you want to provide?
- Several comments left in style.scss to address concerns

###### What are the relevant tickets?
- Ticket #16 
- Ticket #6

### CODE REVIEW

File: src/css/_variables.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+$background-grey: #4d4e4f;
```
Comment (JoshSevy) [inline src/css/_variables.scss:1]: Looks good. Glad we had that lesson today. haha

File: src/css/style.scss [hunk 2]
```diff
@@ -214,69 +214,55 @@ table {
...
-  font-family: sans-serif;
-  display: flex;
-  flex-direction: column;
-  justify-content: space-between;
-  width: 80%;
 }
 
-.card-today-activity {
-  text-align: center;
+.activity-card {
```
Comment (JoshSevy) [inline src/css/style.scss:222]: Love the use of extend, great work!

File: src/index.html [hunk 3]
```diff
@@ -17,28 +17,28 @@ <h2 id="app-message">A message for the user</h2>
...
+              Your stair count today:
               <span class="number" id="flightsOfStairs">0</span>
               Your minutes active today:
               <span class="number" id="minutesActive">0</span>
             </p>
             <button id="new-activity">add new info</button>
           </section>
           <section class="card" id="sleep-today">
-            <p class="card-today-sleep">
+            <p class="sleep-card">
```
Comment (JoshSevy) [inline src/index.html:41]: These are good naming conventions, great work. I actually know what it is doing now. Good Job!

Comment (GreysonElkins) [reply src/index.html:41]: I agree! This is way more manageable. I do think eventually we'll want the card section to have a secondary class (maybe just sleep) and to cut the <p> tags from card elements. This is a great first step!

File: src/css/style.scss [hunk 4]
```diff
@@ -54,34 +48,40 @@ h2 {
...
+  // .sidebar-header-name {
+  //   text-align: center;
+  // }
+// }
 
-.sidebar-header-userInfo {
+/*!! This is being used in 9 different places in page-manip.
+Why is it the header for 'Friends'?
+Come back to this !!*/
+.sidebar-friendslist-header {
```
Comment (GreysonElkins) [inline src/css/style.scss:59]: Personally, I still find a class-name like this a bit confusing. In my head headers and sidebars are different things. Would this be useable if it was just `friendsList`? Especially given that it's being used in so many places

File: src/css/style.scss [hunk 5]
```diff
@@ -308,7 +294,9 @@ table {
   padding-left: 0;
   font-size: 1em;
 }
-
+/* This controls the friends list on the left-sidebar.
+As of right now, I can't see any other use for it.
+Likely needs its name changed.*/
 .historical-list-listItem {
```
Comment (GreysonElkins) [inline src/css/style.scss:300]: For sure agree with you on the name change here Dougie - the weekly calendar also uses the word `historical` for class names, and I think that one could be more specific too. in any case there needs to be some specialization between the two.

**Pull request merged**